newparp/templates/base.html

```diff
@@ -126,6 +127,7 @@ <h2 class="announcements_link"><span class="announce_label">Announcements</span>
 {% endif %}
 <script src="/static/js/ui.js"></script>
 <script src="/static/js/newparp.js"></script>
+<script src="/static/js/zxcvbn.js"></script>
```

Comment:
Putting this in base.html will include the file on every page. Given its size and the fact that it's only used on one page it should go in reset_password.html instead.

Reply:
Alright, I can move that to the reset_password HTML file the only reason I added it to the base was that I planned on adding it to the register page and form, but now thinking about it I can simply add that line to those pages respectively.